cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -165,11 +172,7 @@ var args = []string{
 }
 
 func TestAddFlags(t *testing.T) {
-	fs := pflag.NewFlagSet("addflagstest", pflag.ContinueOnError)
-	s, _ := NewKubeControllerManagerOptions()
-	for _, f := range s.Flags([]string{""}, []string{""}, nil).FlagSets {
-		fs.AddFlagSet(f)
-	}
+	fs, s := setupControllerMgrFlagset(t)
```

Comment:
nit (naming conventions)

```suggestion
	fs, s := setupControllerManagerFlagSet(t)
```

Reply:
thanks, fixed.